tendrl/monitoring_integration/graphite/__init__.py

```diff
@@ -427,7 +427,7 @@ def set_brick_path(self, cluster_data):
             for brick in cluster["Brick"]:
                 try:
                     brick["brick_name"] = brick["brick_path"].split(":")[1]
-                except (AttributeError, KeyError) as ex:
+                except (AttributeError, KeyError, IndexError) as ex:
                     logger.log(
```

Comment:
i saw IndexError when we delete some bricks in volumes. because during brick deletion we are removing all some volumes and sync will populate again. Because subvolume structure may be changed, So while deleting all subvolume this place sometime raise an error while reading etcd.

Reply:
Ok